niji/routes/api/index.js

```diff
@@ -0,0 +1,152 @@
+var express = require('express');
```

Comment:
If you're going to have an api folder, you should be splitting your routes per model and then combining them into index.js.